src/Ajax.php

```diff
@@ -371,7 +371,11 @@ public static function createTabs(
                      tab: index,
                      withtemplate: " . (int)($_GET['withtemplate'] ?? 0) . "
                   }
-               );
+                );
+
+                if (location.hash) {
+                    location.href = location.hash; // move to hanchor if needed
+                }
```

Comment:
This is almost OK, but anchor following should be done only on first load. Indeed, if user goes to another tab and then go back on main tab, this will be evaluated again, and user will loose the position where it previously scrolled.

Reply:
Last commit remove anchor from history to avoid reusing it